ls1intum/Artemis
# `Development`: Reduce transferred data when creating ratings

### Checklist
#### General
#### Server

### Motivation and Context
Creating and updating ratings resulted in a large response from the server. The whole path `result -> participation -> exercise -> course` was included, but this information was not needed on the client.

### Description
I changed the endpoint to only return the star rating integer.

For instructors, information about the participation and exercise are needed on the client, so I only removed the submission und course here.

### Steps for Testing
- As a student, create and update a rating. Make sure everything works
- As an instructor, make sure that the list of ratings works and shows all needed information (assessor, exercise, etc)

### Review Progress

#### Performance Review
#### Code Review
#### Manual Tests
